middleman.js

```diff
@@ -352,18 +352,26 @@ const autofill = async (page, distilled) => {
   return document.documentElement.outerHTML;
 };
 
-const autoclick = async (page, distilled) => {
+const clicks = async (page, distilled, attrs) => {
   const document = parse(distilled);
-  const buttons = document.querySelectorAll('[gg-autoclick]');
+  const buttons = document.querySelectorAll(attrs);
   for (const button of buttons) {
     const { selector, frame_selector } = get_selector(button.getAttribute('gg-match'));
     if (selector) {
-      console.log(`${CYAN}${ARROW} Auto-clicking ${NORMAL}${selector}`);
+      console.log(`${CYAN}${ARROW} Clicking ${NORMAL}${selector}`);
       await click(page, selector, 3 * 1000, frame_selector);
     }
   }
 };
 
+const autoclick = async (page, distilled) => {
+  await clicks(page, distilled, '[gg-autoclick]');
+};
+
+const autosubmit = async (page, distilled) => {
+  await clicks(page, distilled, '[type="submit"]');
+};
+
```

Comment:
This is a very neat trick!
And while you're there, might as well get rid of both functions since they are small enough and the caller can simply call `clicks` with the right `attrs`.